apps/course/src/app/features/legal/legal-detail/legal-detail.component.ts

```diff
@@ -0,0 +1,55 @@
+import { Component, OnInit } from '@angular/core';
+import { LegalInfoComponent } from './legal-info/legal-info.component';
+import { LegalMarkdownRendererComponent } from './legal-markdown-renderer/legal-markdown-renderer.component';
+import { Legal } from '../../../types/legal.type';
+import { ActivatedRoute, Router } from '@angular/router';
+import { LegalService } from '../../../core/legal.service';
+import { Meta, Title } from '@angular/platform-browser';
+import { environment } from '../../../../environments/environment';
+import { AppComponent } from '../../../app.component';
+
+@Component({
+  selector: 'app-legal-detail',
+  standalone: true,
+  imports: [LegalInfoComponent, LegalMarkdownRendererComponent],
+  templateUrl: './legal-detail.component.html',
+  styleUrl: './legal-detail.component.scss',
+})
+export class LegalDetailComponent implements OnInit {
+  public legal: Legal | undefined;
+
+  constructor(
+    private route: ActivatedRoute,
+    private router: Router,
+    private legalService: LegalService,
+    private title: Title,
+    private metaService: Meta,
+  ) {}
+
+  ngOnInit(): void {
+    const legalId = this.route.snapshot.paramMap.get('legalId');
+    if (legalId) {
+      this.legal = this.legalService.getLegalByFile(legalId);
+      if (this.legal) {
+        {
+          this.title.setTitle(
+            this.legal.name +
+              ' - ' +
+              environment.metaConfig.title +
+              ' - ' +
+              AppComponent.chorizo.title,
+          );
+          this.metaService.updateTag({
```

Comment:
Isn't needed yet and has to be implemented otherwise. Can be removed 👍🏻

Reply:
What exactly isn't needed yet?

Reply:
`metaService.updateTag({..`

Reply:
so I can delete this peace of code:
this.metaService.updateTag({
            name: 'description',
            content: this.legal.description,
          });

Reply:
Yes